contributors.md

```diff
@@ -172,8 +200,12 @@ next step for The Turing Way to become more than a book. It could be an
 ecosystem, with the book at its basis. And -- if we decide to go that route --
 I would like to be a part of it.
 
+<!---I--->
+
+
 ### Ismael Kherroubi Garcia
 
+* Role: Core contributor, [OLS project lead](https://github.com/alan-turing-institute/the-turing-way/tree/master/open-life-science-mentoring): Ethics Research Framework
```

Comment:
Is being part of OLS a role? I feel like its a pathway to engagement rather than a role....

Reply:
"OLS project lead" is a role.